app/views/Home.scala.html

```diff
@@ -246,31 +247,54 @@ <h1 class="govuk-heading-xl @if(user.saUtr.isDefined) { margin-bottom-zero }">
 }
 
 @paymentHistoryTile = {
-@home_tile("payment-history-tile", messages("home.paymentHistory.heading")) {
-    @p(classes = "govuk-!-margin-bottom-4") {
-            @link(
-                link = paymentHistoryControllerURL,
-                messageKey = "home.paymentHistory.view"
-
-            )
-    }
-
-    @if(creditAndRefundEnabled) {
-        @if(isUserMigrated) {
-            @link(
-                link = creditAndRefundsControllerURL,
-                messageKey = "home.credAndRefund.view"
-            )
-        } else {
-            @link(
-                link = creditAndRefundsControllerURL,
-                messageKey = "home.credAndRefund_howToClaim.view"
-            )
-        }
-    }
-}
+    @if(paymentHistoryEnabled) {
+             @home_tile("payment-history-tile", messages("home.paymentHistoryRefund.heading")) {
```

Comment:
please fix all formatting here - should be 4 spaces for a tab

Reply:
Hi,Fixed the formatting issues.